docs/generated/packages/next/documents/next-config-setup.md

```diff
@@ -1,132 +0,0 @@
----
-title: How to configure Next.js plugins
-description: A guide for configuring Next.js plugins with Nx
----
-
-# Configuring Next.js plugins
-
-Next.js plugins are configured in the project's `next.config.js` file. Nx adds a its own plugin (`withNx`) to help the Next.js application
-understand workspace libraries, and other Nx-specific features. See below for an example of the `withNx` plugin that Nx adds by default.
-
-```js
-// next.config.js
```

Comment:
let's use the `{% fileName...` here (in the original file, not this deleted one)